python/mlcroissant/mlcroissant/_src/core/rdf_test.py

```diff
@@ -1,16 +1,18 @@
 """Tests for RDF."""
 
+from mlcroissant._src.core.context import Context
 from mlcroissant._src.core.rdf import Rdf
 
 
 def test_shorten_value():
     rdf = Rdf.from_json(
+        ctx=Context(),
```

Comment:
Would it make sense to add a test for when ctx is not empty?

Reply:
Good idea, thanks!